src/core/fonts.js

```diff
@@ -722,6 +722,11 @@ function createPostTable(properties) {
   ); // maxMemType1
 }
 
+function makePostscriptName(name) {
```

Comment:
Nit: I'd probably call this `createPostscriptName` for improved consistency, since all of the surrounding functions use that pattern.